### ISSUE TITLE
get accessors only transform the first one in a class

### ISSUE THREAD

**Issue by pilaoda** [TypeScriptToLua/TypeScriptToLua#1590]
I can't reproduce it in the playground, and I guess it may be relevant to the legacy decorator. 
Don't know how to config the playground to use legacy decorator

https://github.com/TypeScriptToLua/TypeScriptToLua/blob/e10839386c17b0900bfccb4ace9dc45f523f33f0/src/transformation/visitors/class/index.ts#L181
I debug my local project and found it was caused by this line `continue`. What's the purpose of this code?

**Comment by pilaoda** [comment]
```
export class CG {
	get a() {
		return 1;
	}
	get b() {
		return 2;
	}
}
```

only output:
```
function CG.prototype.____constructor(self)
end
__TS__SetDescriptor(
    CG.prototype,
    "a",
    {get = function(self)
        return 1
    end},
    true
)
```

This happen in my local project

**Comment by pilaoda** [comment]
found it fix in 1.26.2
